Code/Tools/TestImpactFramework/Runtime/Common/Code/Source/Artifact/Factory/TestImpactTestTargetMetaMapFactoryUtils.cpp

```diff
@@ -25,6 +25,7 @@ namespace TestImpact
             labelSet.insert(labelString);
         }
 
-        return labelSet;
+        // Only test suite labels that contain the TIAF requirement label
+        return labelSet.contains(RequiresTiafLabel) ? std::optional<SuiteLabelSet>{ labelSet } : AZStd::nullopt;
```

Comment:
At the very least the std::optional should be `AZStd::optional`, especially since the second part of the ternary is using **AZStd**::nullopt
```suggestion
        return labelSet.contains(RequiresTiafLabel) ? AZStd::optional<SuiteLabelSet>{ labelSet } : AZStd::nullopt;
```